lib/screens/authentication/recover/recover_account_search/interactor/mappers/fetch_account_guardians_state_mapper.dart

```diff
@@ -4,24 +4,23 @@ import 'package:seeds/domain-shared/result_to_state_mapper.dart';
 import 'package:seeds/i18n/authentication/recover/recover.i18n.dart';
```

Comment:
Are these changes related to the PR?

Reply:
Nope but this class name did not match its file name.